src/pages/ResetPassword.tsx

```diff
@@ -100,9 +100,10 @@ export const ResetPassword = () => {
               <ul>
                 <li>at least 12 characters long,</li>
                 <li>
-                  a combination of uppercase letters, lowercase letters,
-                  numbers, and symbols.
+                  a combination of uppercase letters, lowercase letters and
+                  numbers
                 </li>
+                <li>at least one symbol from: <span style={{fontFamily: 'monospace'}}>! @ # $ % ^ &amp; *</span></li>
```

Comment:
Ideally, we should create a CSS style instead of inline for this and apply it using a class.

Reply:
addressed in c9d320bd9e113ad5d2d96e81718194a6f19e8944
Thank you for the review!